deployment/tfhub/object-detection/base64/object-detection-base64-deploy.ipynb

```diff
@@ -0,0 +1,390 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook has been tested with Python 3.10.6 and the following package versions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install dill==0.3.6\n",
+    "!pip install Pillow==9.3.0\n",
+    "!pip install tensorflow==2.11.0\n",
+    "!pip install tensorflow-hub==0.12.0\n",
+    "!pip install urllib3==1.26.13\n",
+    "!pip install verta==0.21.1\n",
+    "!pip install wget==3.2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import base64\n",
+    "import io\n",
+    "import json\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import tempfile\n",
+    "import tensorflow as tf\n",
+    "import tensorflow_hub as hub\n",
+    "import time\n",
+    "import wget\n",
+    "\n",
+    "from PIL import Image, ImageOps\n",
+    "from verta import Client\n",
+    "from verta.endpoint.autoscaling import Autoscaling\n",
+    "from verta.endpoint.autoscaling.metrics import CpuUtilizationTarget, MemoryUtilizationTarget, RequestsPerWorkerTarget\n",
+    "from verta.endpoint.resources import Resources\n",
+    "from verta.endpoint.update import DirectUpdateStrategy\n",
+    "from verta.environment import Python\n",
+    "from verta.registry import VertaModelBase, verify_io\n",
+    "from verta.utils import ModelAPI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Model Class"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class DetectObject(VertaModelBase):\n",
+    "    def __init__(self, artifacts=None):\n",
+    "        module_handle = 'https://tfhub.dev/google/openimages_v4/ssd/mobilenet_v2/1'\n",
+    "        self.detector = hub.load(module_handle).signatures['default']\n",
+    "\n",
+    "        url = 'http://s3.amazonaws.com/verta-starter/street-view-images/000001_0.jpg'\n",
+    "        file = url.split('/')[-1]\n",
+    "        wget.download(url, file)\n",
+    "\n",
+    "        with open(file, 'rb') as img:\n",
+    "            img_bytes = base64.b64encode(img.read())\n",
+    "            img_str = img_bytes.decode('utf-8')\n",
+    "            img_str = json.dumps(img_str)\n",
+    "            img_str = np.array(img_str).tolist()\n",
+    "        \n",
+    "        self.img_base64 = img_str\n",
```

Comment:
I think this (from `url = ...` to `self.img_base64`) could be moved into `def example()`